.vscode/settings.json

```
@@ -10,5 +10,13 @@
   "autoDocstring.includeName": true,
   "autoDocstring.startOnNewLine": true,
   "editor.tabCompletion": "on",
-  "editor.snippetSuggestions": "inline"
+  "editor.snippetSuggestions": "inline",
+  "conventional-branch.type": [
+    "docs",
+    "feat",
+    "fix",
+    "build",
+    "chore"
+  ],
```

Not sure if it overcomplicates things, but we could add "ci", "refactor", "style", and "test"?

```suggestion
    "docs",
    "feat",
    "fix",
    "build",
    "chore",
    "ci",
    "refactor",
    "style",
    "test"
];
```

In addition, we could add what they cover in a comment (just until we all get them under our belt)?
- "build",  # Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
- "ci", # Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
- "docs", #Documentation only changes
- "feat", #A new feature
- "fix", #A bug fix
- "refactor",#A code change that neither fixes a bug nor adds a feature
- "style", #Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
- "test", #Adding missing tests or correcting existing tests

---

I thought of that too, but do we mimic *exactly* what is in conventional commits? The conventional commits makes sense at the commit level, but the branch level, having a branch for "just tests" make sense?

---

Not sure. In a scenario where someone forgot to add tests to some new functionality, or if we decide we need additional testing for something that is already done, then maybe yes?

---

I'm not that passionate about adding these extra prefixes. We can also just omit them for now, and then add them late on, if we find that they would be useful